tests/libnet/cloudinit.go

```diff
@@ -38,15 +38,21 @@ type CloudInitInterface struct {
 	Gateway6       string               `json:"gateway6,omitempty"`
 	Nameservers    CloudInitNameservers `json:"nameservers,omitempty"`
 	MACAddress     string               `json:"macaddress,omitempty"`
+	Match          CloudInitMatch       `json:"match,omitempty"`
 	MTU            int                  `json:"mtu,omitempty"`
 	Routes         []CloudInitRoute     `json:"routes,omitempty"`
+	SetName        string               `json:"set-name,omitempty"`
 }
 
 type CloudInitNameservers struct {
 	Search    []string `json:"search,omitempty,flow"`
 	Addresses []string `json:"addresses,omitempty,flow"`
 }
 
+type CloudInitMatch struct {
+	MACAddress string `json:"macaddress,omitempty"`
+}
```

Comment:
Maybe you should add the other available options.
But this is not a must, whatever you prefer.

Reply:
Done